### ISSUE TITLE
ProcessorScheduler method comment refers to DelayScheduler class comment

### ISSUE THREAD

**Issue by StephanEggermont** [guillep/pharo-core#2282]
In DelayScheduler there used to be a class comment containing a.o. 
----
Original word of advice:
DelayScheduler is THE highest priority code which is run in Pharo.  In other words it is time-critical. The speed of this code is critical for accurate responses, it is critical for network services, it affects every last part of the system. Don't fix it if it ain't broken! This code isn't supposed to be beautiful, it's supposed to be fast! The reason for duplicating code is to make it fast. The reason for not using ifNil:[]ifNotNil:[] is that the compiler might not inline those. Since the effect of changes are VERY hard to predict it is best to leave things as they are for now unless there is an actual need to change anything.

Revised word of advice:
Now you can swap between different schedulers on the fly, have a go. Create your own subclass of  DelayScheduler and use DelayBenchmark to compare the alternatives.  
----
This is referred to in ProcessorScheduler>>timingPriority

Also port back to Pharo 7

**Comment by bencoman** [comment]
I should have know better but couldn't find "DelayScheduler" class when I looked for it. duh! in the transition to the cleaned hierarchy it was awkward to co-exist so the new one became DelayBasicScheduler.  So side question... Now the old DelayScheduler is gone, would it be better for DelayBasicScheduler to be renamed DelayScheduler? (but not for Pharo 7)  Or just refer to "delay scheduler" instead of "DelayScheduler"?

**Comment by bencoman** [comment]
Not sure which method or class comment you intend your "revised word of advice" to update.
The "original word of advice" was moved to DelayBenchmark.  We could discuss on-the-fly delay scheduler swapping there and update comment here to just be...
```
timingPriority
    	"Answer the priority at which the system processes keeping track of real time should run."
	"BEFORE USING IT: please read DelayBenchmark class comment. 
	TL;DR: for top priority processes use #highestNonTimingPriority except if you really know what you are doing!"
```
where...
```
highestNonTimingPriority 
    ^ self timingPriority - 1
```

**Comment by bencoman** [comment]
To avoid more side discussion here, I added... #2287
